python/pickles-in-numpy.yml

```diff
@@ -0,0 +1,8 @@
+rules:
+- id: pickles-in-numpy
+  pattern: numpy.load(..., allow_pickle=True)
```

Comment:
1) Handle `allow_pickle=1` or `allow_pickle=<non zero int>` if they work
2) Handle or document `**kwargs`

Reply:
The rule has been updated to handle 1. It catches case 2.